tests/regionizers/test_h3_regionizer.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for H3Regionizer."""
+
+import osmnx as ox
+import pytest
+
+from srai.regionizers.h3_regionizer import H3Regionizer
```

Comment:
With added import to `srai/regionizers/__init__.py` this can be shortened.
```suggestion
from srai.regionizers import H3Regionizer
```